ERP-BACKEND/app/routers/crm.py

```
@@ -1,108 +1,23 @@
-    probability: int
-    expected_close_date: Optional[date] = None
-    actual_close_date: Optional[date] = None
-    assigned_to: Optional[int] = None
-    description: Optional[str] = None
-    created_at: datetime
-    updated_at: datetime
-
 # Companies
 @router.post("/companies", response_model=CompanyResponse)
-def create_company(data: CompanyCreate, db: Session = Depends(get_db), current_user = Depends(get_current_user)):
+def create_company(data: CompanyUpdate, db: Session = Depends(get_db), current_user = Depends(get_current_user)):
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)